src/marine/argoClim2ioda.py

```diff
@@ -67,9 +67,12 @@ def _readData(self):
         assert self.varname in ['TEMPERATURE', 'SALINITY'],\
             "%s is not a valid variable name" % self.varname
 
+        self.varname2 = 'waterTemperature'
+        
         lon = nc.variables['LONGITUDE'][:]
         lat = nc.variables['LATITUDE'][:]
-        pres = nc.variables['PRESSURE'][:]
+        pres = nc.variables['PRESSURE'][:] #pressure in decibar
+        depth = pres   #1 decibar = 1 meter, so you can use pressure as depth
 
         # Get absolute time instead of time since epoch
         dtime = nc.variables['TIME']
```

Comment:
The input file ```testinput/argoclim_test.nc``` already has the variable ```TIME``` with an attribute for ```units``` set to ```months since 2004-01-01 00:00:00```.  IODA cannot really handle the ```months since``` yet and would really need ```seconds since``` but it could be possible to bypass a lot of date manipulation happening inside this code with some potential manipulations of the Epoch and values in ```TIME``` to conform to IODA.  Perhaps we can leave this alone right now, but I see potential for beneficial changes in this area.